packages/frontend/src/pages/ParkingSpace/index.tsx

```diff
@@ -41,8 +41,11 @@ const ParkingSpaceTabs = (props: { listingId: number }) => {
   })
 
   const [selectedTab, setSelectedTab] = useState(() => {
-    if (data.status === ListingStatus.Assigned && data.offers.length > 0) {
-      return String(data.offers[0].id)
+    if (
+      data.status === ListingStatus.Assigned ||
+      (data.status === ListingStatus.Expired && data.offers.length > 0)
+    ) {
+      return String(data.offers[data.offers.length - 1].id)
```

Comment:
Would it be sufficient to just check for offers.length?
Is there any case where there are offers but we don't want to show the latest offer round first?

Reply:
Yes! I think you're right. Just gonna verify by some testing.